content/blog/2022/02/2022-06-28-require-java-11.adoc

```diff
@@ -0,0 +1,323 @@
+---
+layout: post
+title: "Jenkins requires Java 11"
+tags:
+- announcement
+- core
+- developer
+- jenkins
+- platform-sig
+author: basil
+description: >
```

Comment:
should have an opengraph image I expect, there's a deck here with examples:
https://docs.google.com/presentation/d/1Q1PgNnRTgzBpVRXPqQo3PudzCa2eoc6_1_NRjFRMLrU/edit#slide=id.g8b90766921_1_0